pages/community/code-of-conduct.md

```diff
@@ -0,0 +1,122 @@
+may all experience some frustration from time to time, but we do not allow frustration to turn into a personal attack. A community where
+people feel uncomfortable or threatened is not a productive one.  
+* Be collaborative. Our work depends on the participation of many people, and in turn others depend on our work. Effective and friendly collaboration enables everyone to better achieve their goals.  
+* Be careful in the words that we choose. We are careful and respectful in our communication, and we take responsibility for our own speech. Be kind to others. Do not insult or put down other members of the community.
+
+<br>
+
+## Unacceptable Behavior
+We, the members of the US-RSE Association, are committed to making participation in this community a harassment-free experience.
+
+We will not accept harassment or other exclusionary behaviors, such as:
+* The use of sexualized language or imagery  
+* Excessive profanity (please avoid curse words; people differ greatly in their sensitivity to swearing)  
+* Posting sexually explicit or violent material
+* Violent or intimidating threats or language directed against another person or group
+* Inappropriate physical contact and/or unwelcome sexual attention or sexual comments
+* Sexist, racist, or otherwise discriminatory jokes and language
+* Trolling or insulting and derogatory comments
+* Written or verbal comments which have the effect of excluding people on the basis of membership in a specific group, including level
+of experience, gender, gender identity and expression, sexual orientation, disability, neurotype, personal appearance, body size, race,
+ethnicity, age, religion, or nationality
+* Public or private harassment
+* Sharing private content, such as emails sent privately or non-publicly, or direct message history, without the sender’s consent
+* Continuing to initiate interaction (such as photography, recording, messaging, or conversation) with someone after being asked to stop
+* Sustained disruption of talks, events, or communications, such as heckling of a speaker
+* Publishing (or threatening to post) other people’s personally identifying information (“doxing”), such as physical or electronic
+addresses, without explicit permission
+* Other unethical or unprofessional conduct
+* Advocating for, or encouraging, any of the above behaviors
+
+<br>
+
+## Reporting Guidelines
+If you believe someone is violating the code of conduct, please report this in a timely manner (see How to Submit a Report). Code of
+conduct violations reduce the value of the community for everyone. The US-RSE Association members take reports of misconduct very seriously
+and are committed to preserving and maintaining the welcoming nature of our community.
+
+All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the
+circumstances. All reports will be kept confidential.
```

Comment:
What about cases which need to be reported to police? "All reports will be kept confidential, with the exception of cases where the steering committee determines it prudent to share the report with law enforcement. In those cases, the report will be shared only with the proper legal authorities."

Reply:
Yep that would be good to add. Lord, I hope it never comes to that.

Reply:
Agreed. Worth an add.  Personally speaking, I can't imagine using the CoC as the reason I didn't report something to police.

Reply:
Added. I hope it never comes to that either. But good to have the option stated clearly.